src/python/twitter/common/reviewboard/reviewboard.py

```diff
@@ -103,7 +103,7 @@ def _add_auth_params(self, username, password):
     urllib2.install_opener(opener)
 
   def get_url(self, rb_id):
-    return '%s/r/%s' % (self.url, rb_id)
+    return '%sr/%s' % (self.url, rb_id)
```

Comment:
-1.
%sr is meaningless. In case, if self.url is sent without trailing '/'. This will break. Instead, having // seems better as it can cause no harm.